2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,2404 @@
+      summary: Get a Sample
+  /studies/{study_id}:
+    get:
+      description: Given a Study ID, get the Study record from the metadata store.
+      operationId: get_studies_studies__study_id__get
+      parameters:
+      - in: path
+        name: study_id
+        required: true
+        schema:
+          title: Study Id
+          type: string
+      - in: query
+        name: embedded
+        required: false
+        schema:
+          default: false
+          title: Embedded
+          type: boolean
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Study'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Get a Study
+  /submissions:
+    post:
+      description: 'Add a Submission object to the metadata store, where the Submission
+        object
+
+        represents a Study with one or more Experiments, Analyses, Samples, Files,
+        etc.'
+      operationId: add_submission_submissions_post
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Submission'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Submission'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Add a Submission
+  /submissions/{submission_id}:
+    get:
+      description: Given a Submission ID, get the corresponding Submission object.
+      operationId: get_submission_submissions__submission_id__get
+      parameters:
+      - in: path
+        name: submission_id
+        required: true
+        schema:
+          title: Submission Id
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Submission'
+          description: Successful Response
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      summary: Given a Submission ID, get the corresponding submission
+    patch:
+      description: Update status of a Submission.
+      operationId: update_submission_status_submissions__submission_id__patch
+      parameters:
+      - in: path
+        name: submission_id
+        required: true
+        schema:
+          title: Submission Id
+          type: string
+      - in: query
```

Comment:
The new status shouldn't be set via query parameter but it should be specified in the body.